mkdocs.yml: add links of social media

Domarys opened the pull request.

otavio commented on mkdocs.yml:
```diff
@@ -1,6 +1,31 @@
+
+
+# Copyright
+copyright: 'Copyright &copy; 2019 O.S.System'
```
The company name is `O.S. Systems`

gustavosbarreto replied:
The year is 2020

Domarys replied:
fixed

otavio commented on mkdocs.yml:
```diff
@@ -9,7 +9,7 @@ repo_url: 'https://github.com/shellhub-io/shellhub-io.github.io'
 
 # Copyright
-copyright: 'Copyright &copy; 2019 O.S.System'
+copyright: 'Copyright &copy; 2019 O.S. System'
```
@Domarys the other minor thing is the year. We are past 2019 already ;-)

Domarys committed:
mkdocs: rework and add infos

include links social media, Project information and standard reference: https://squidfunk.github.io/mkdocs-material/getting-started/

otavio commented on mkdocs.yml:
```diff
@@ -36,3 +57,10 @@ markdown_extensions:
+  - What is ShellHub?: index.md
+  - Introduction: introduction.md
+  - Getting started: getting-started.md
+  
```
There is no newline at end of file. Likely previous line has a spurious space or something like that.

otavio commented on mkdocs.yml:
```diff
@@ -63,4 +63,4 @@ nav:
   - Getting started: getting-started.md
-  
\ No newline at end of file
+
```
This commit should be squashed on previous one.
